spec/javascripts/fixtures/NewsletterSticky.html

```diff
@@ -1,9 +1,11 @@
 <div id="fixture-1">
   <div class="news-signup-sticky news-signup-sticky--positioned" data-dough-component="NewsletterSticky">
     <form onsubmit="return false;">
-      <button class="news-signup-sticky__close">
+      <button class="news-signup-sticky__close" id="first-focusable">
```

Comment:
That's a little risky for an ID, if this test pulled in multiple fixtures both having that ID... Albeit it's a very slim chance so it's debatable.